pandas/core/base.py

```diff
@@ -877,6 +877,10 @@ def _values(self):
         """ the internal implementation """
         return self.values
 
+    @property
+    def empty(self):
+        return not bool(self.size)
```

Comment:
t looks like Series is inheriting its current implementation from `NDFrame`, but this one will win because `IndexOpsMixin` appears first in its bases.